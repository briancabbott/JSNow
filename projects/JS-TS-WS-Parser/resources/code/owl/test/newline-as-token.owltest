a foo
a bar
---
.whitespace ' ' '\t'
program = (line? '\n')*
line = 'a' identifier
