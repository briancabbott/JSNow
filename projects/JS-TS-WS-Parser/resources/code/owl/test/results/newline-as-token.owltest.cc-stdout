program (0 - 12)
  line (0 - 5)
    identifier - foo (2 - 5)
  line (6 - 11)
    identifier - bar (8 - 11)
