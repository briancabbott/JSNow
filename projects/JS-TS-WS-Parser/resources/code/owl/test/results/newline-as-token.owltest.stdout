. a foo
  line-

. a bar
  line-
